doc/OVERMAP.md

```diff
@@ -0,0 +1,426 @@
+
+### Fields
+
+|   Identifier    |                                              Description                                              |
+| --------------- | ----------------------------------------------------------------------------------------------------- |
+| `type`          | Must be "overmap_special".                                                                            |
+| `id`            | Unique id.                                                                                            |
+| `overmaps`      | List of overmap terrains and their relative location within the special. Location is [ x, y, z ].     |
+| `connections`   | List of overmap connections and their relative location within the special. Location is [ x, y, z ].  |
+| `locations`     | List of `overmap_location` ids that the special may be placed on.                                     |
+| `city_distance` | Min/max distance from a city that the special may be placed. Use -1 for unbounded.                    |
+| `city_sizes`    | Min/max city size for a city that the special may be placed near. Use -1 for unbounded.               |
+| `occurrences`   | Min/max number of occurrences when placing the special. If UNIQUE flag is set, becomes X of Y chance. |
+| `flags`         | See `Overmap specials` in JSON_FLAGS.md.                                                               |
+| `rotate`        | Whether the special can rotate. True if not specified.                                                |
+
+### Example
+
+```json
+[
+  {
+    "type": "overmap_special",
+    "id": "campground",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "campground_1a_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "campground_1b_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "campground_2a_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "campground_2b_north" }
+    ],
+    "connections": [ { "point": [ 1, -1, 0 ], "terrain": "road", "connection": "local_road", "from": [ 1, 0, 0 ] } ],
+    "locations": [ "forest" ],
+    "city_distance": [ 10, -1 ],
+    "city_sizes": [ 3, 12 ],
+    "occurrences": [ 0, 5 ],
+    "flags": [ "CLASSIC" ],
+    "rotate" : true
+  }
+]
+```
+
+### Connections
+
+|  Identifier  |                                           Description                                            |
+| ------------ | ------------------------------------------------------------------------------------------------ |
+| `point`      | [ x, y, z] of the connection end point. Cannot overlap an overmap terrain entry for the special. |
+| `terrain`    | Will go away in favor of `connection` eventually. Use `road`, `subway`, `sewer`, etc.            |
+| `connection` | Id of the `overmap_connection` to build. Optional for now, but you should specify it explicitly. |
+| `from`       | Optional point [ x, y, z] within the special to treat as the origin of the connection.           |
+
+## City Building
```

Comment:
Note that they don't contribute to overmap specials limit and number of their occurrences cannot be controlled other than by regional settings?

Reply:
I was attempting to do that in what I wrote [here](https://github.com/CleverRaven/Cataclysm-DDA/pull/28845/files#diff-aac95b3bbc78b6180d359b1327c90610R319). Do you think it needs to be written differently?

Reply:
Looks good.